adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/Binary.kt

```diff
@@ -90,7 +90,7 @@ class Binary {
         result = 31 * result + architecture.hashCode()
         result = 31 * result + image_type.hashCode()
         result = 31 * result + jvm_impl.hashCode()
-        result = 31 * result + (`package`?.hashCode() ?: 0)
+        result = 31 * result + `package`.hashCode()
```

Comment:
why the back ticks?

Reply:
package is a reserved word in kotlin same as java